assets/i18n/ar.json

```diff
@@ -366,6 +367,13 @@
 	"I18N_EXPLORATION_zW39GLG_BdN2_DESCRIPTION": "بينما يتعلم ماثيو كيفية مقارنة الكسور من حيث الحجم ، يقع حادث في المخبز ، وينزعج السيد بيكر. دعونا نرى ما حدث!",
 	"I18N_EXPLORATION_zW39GLG_BdN2_TITLE": "مقارنة الكسور",
 	"I18N_FEEDBACK_MESSAGE_ANONYMOUS_AUTHOR": "مجهول",
+	"I18N_FEEDBACK_UPDATES_FEEDBACK_THREAD_WARNING": "تجنب مشاركة أية معلومات شخصية نظرا لأن هذه المناقشة قابلة للعرض بشكل عام.",
+	"I18N_FEEDBACK_UPDATES_RETURN_TO_FEEDBACK_THREADS_MESSAGE": "ارجع إلى قائمة الرسائل",
+	"I18N_FEEDBACK_UPDATES_SEND_FEEDBACK_THREAD_MESSAGE": "أرسل",
+	"I18N_FEEDBACK_UPDATES_SEND_FEEDBACK_THREAD_MESSAGE_IN_PROGRESS": "جارٍ الإرسال...",
+	"I18N_FEEDBACK_UPDATES_THREAD_DEFAULT_MESSAGE": "الرد",
+	"I18N_FEEDBACK_UPDATES_THREAD_STATUS_CHANGE_MESSAGE": "تم تغيير الحالة إلى '<[threadStatus]>'",
+	"I18N_FEEDBACK_UPDATES_VIEW_SUGGESTION": "عرض الاقتراح",
```

Comment:
Hmm, does this naming convention looks good? I would like to take a second opinion as previously we were using `LEARNER_DASHBOARD` as a suffix because the functionality was present on learner-dashboard page and currently we are adding a suffix as the name of the functionality. Otherwise similar comments on the codeowners files as @seanlip  and @kevintab95.
cc @seanlip @kevintab95
Thanks

Reply:
I think it's ok, I think of this as being "stuff relating to the 'feedback updates' component". Do you have a better name to suggest @lkbhitesh07?